include/seqan3/alphabet/composite/alphabet_tuple_base.hpp

```diff
@@ -144,18 +144,17 @@ decltype(auto) get();
 
 /*!\brief The CRTP base for a combined alphabet that contains multiple values of different alphabets at the same time.
  * \ingroup composite
- * \implements seqan3::Semialphabet
- * \implements seqan3::detail::ConstexprSemialphabet
- * \tparam first_component_type Type of the first letter; must model seqan3::Semialphabet.
- * \tparam component_types      Types of further letters (up to 4); must model seqan3::Semialphabet.
+ * \implements seqan3::WritableSemialphabet
+ * \if DEV \implements seqan3::detail::ConstexprWritableSemialphabet \endif
+ * \tparam component_types Types of letters; must model seqan3::detail::WritableConstexprSemialphabet.
```

Comment:
It looks like you used a detail concept in a public documentation. Was this intended?

Reply:
Yes, because it requires the constexpr-versions. There was a design for also supporting non-constexpr alphabets with a different specialisation, but it was never implemented.

But I guess, I could also require the non-constexpr alphabet and state that all function calls need to be executable in a constexpr context.

Reply:
>Yes, because it requires the constexpr-versions. There was a design for also supporting non-constexpr alphabets with a different specialisation, but it was never implemented.

I assumed that it requires the constexpr-version, but my question was regarding the user API documentation.
In public API it links to nowhere since it is developer API. So also clear in it's name it simply raises the question of what is this. So I find it better to just write that it requires all functions of WritableSemialphabt to be constexpr without linking to the actual developer docu. This could be like above wrapped for the dev docu specifically.

Reply:
Yep, that's what I did now.